x/claim/types/expected_keepers.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	epochstypes "github.com/Stride-Labs/stride/x/epochs/types"
+)
+
+// BankKeeper defines the banking contract that must be fulfilled when
+// creating a x/claim keeper.
+type BankKeeper interface {
+	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
+	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+	SendCoinsFromModuleToModule(ctx sdk.Context, senderPool, recipientPool string, amt sdk.Coins) error
+	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
+	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
```

Comment:
Is there any other way to set up the test? It would be nice to keep the permissions on this module as tight as possible (if it's not possible, that's ok, we should just be careful when reviewing code to the claim / vesting modules)